main/variables.tf

```diff
@@ -0,0 +1,206 @@
+# Global
+
+variable "profile" {
+  description = "AWS Profile to use."
+  type        = "string"
+}
+
+variable "region" {
+  description = "AWS Region name"
+  default     = "us-west-2"
+  type        = "string"
+}
+
+variable "tags" {
+  description = "A map of tags to apply to resources."
+  type        = "map"
+}
+
+# Networking
+
+variable "vpc-name" {
+  description = "Name of the VPC in which to install Beekeeper."
+  type        = "string"
+}
+
+variable "subnet-names" {
+  description = "Names of the subnets in which to install Beekeeper."
+  type        = "list"
+}
+
+variable "security-group-names" {
+  description = "Names of the security groups in which to install Beekeeper ECS."
+  type        = "list"
+}
+
+# RDS specific
+
+variable "subnet-group-name" {
+  description = "RDS subnet group name."
+  default     = "bdp-db-subnet-group"
+  type        = "string"
+}
+
+variable "allocated-storage" {
+  description = "RDS allocated storage."
+  default     = 10
+  type        = "string"
+}
+
+variable "storage-type" {
+  description = "RDS storage type."
+  default     = "gp2"
+  type        = "string"
+}
+
+variable "instance-class" {
+  description = "RDS instance class."
+  default     = "db.t2.micro"
+  type        = "string"
+}
+
+variable "engine-version" {
+  description = "RDS engine version."
+  default     = "8.0"
+  type        = "string"
+}
+
+variable "parameter-group-name" {
+  description = "RDS parameter group name."
+  default     = "default.mysql8.0"
+  type        = "string"
+}
+
+variable "username" {
+  description = "RDS username."
+  default     = "bdp"
+  type        = "string"
+}
+
+variable "password" {
+  description = "RDS password."
+  default     = "Ch4ll3ng3"
```

Comment:
We should not provide default password for open-source projects. Password should always be provided by the user of this module with no defaults.

Reply:
Good idea, this is now removed